client/src/theme/theme.js

```diff
@@ -3,7 +3,7 @@ import { createTheme } from '@mui/material';
 const theme = createTheme({
   breakpoints: {
     values: {
-      xs: 0,
+      zero: 0,
```

Comment:
don't think that it is a good idea to change 'xs' to 'zero'. I suggest leaving it as it was. It will enable us to change xs breakpoint by necessity, and it will be changed everywhere else